docs/asciidoc/modules/ROOT/examples/generated-documentation/apoc.meta.nodes-lite.csv

```diff
@@ -0,0 +1,2 @@
+¦signature
+¦apoc.meta.nodes.count(nodes = [] :: LIST? OF STRING?, config = {} :: MAP?) :: (INTEGER?)
```

Comment:
aren't these files generated and should not be checked in? everything in generated documentation.